.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ commands:
     steps:
       - run:
           name: "Pull submodules"
-          command: git submodule update --init
+          command: git submodule update --init --recursive
```

Comment:
Just wondering, is this `--recursive` a new requirement? (Because it also pulls another submodule that we don't actually need).

Reply:
Without that, the code would not compile because of some missing dependency